tests/common/devices/multi_asic.py

```diff
@@ -204,6 +204,16 @@ def get_linux_ip_cmd_for_namespace(self, cmd, namespace):
         ns_cmd = cmd.replace('ip', 'ip -n {}'.format(namespace))
         return ns_cmd
 
+    @property
+    def ttl_decr_value(self):
+        """
+        Decrement in TTL value for L3 forwarding. On Multi ASIC TTL value
+        decreases by 3 when forwarding across tiers (e.g. T0 to T2).
+        """
+        if not self.sonichost.is_multi_asic:
+            return 1
+        return 3
```

Comment:
I think this is ok for our current multi asic topology. If T0 and T2 are connected to same asic then the ttl will be 1.